tests/basic/commands

```diff
@@ -35,6 +35,10 @@ test tests/basic/undefinedReferences.ref get undefinedReferences
 test tests/basic/uniqueIpAssignments.ref get uniqueIpAssignments
 test tests/basic/unusedStructures.ref get unusedStructures
 
+# example2 testrig
+test tests/basic/init-example2.ref init-testrig test_rigs/example2
+test tests/basic/inferroles.ref get inferRoles
```

Comment:
Or maybe you just mean that inferroles.ref should be inferRoles.ref? That would be more consistent with the other tests.